hudi-utilities/src/test/java/org/apache/hudi/utilities/testutils/UtilitiesTestBase.java

```diff
@@ -410,6 +411,16 @@ public static String toJsonString(HoodieRecord hr) {
       return records.stream().map(Helpers::toJsonString).toArray(String[]::new);
     }
 
+    public static Tuple2<String, String>[] jsonifyRecordsByPartitions(List<HoodieRecord> records, int partitions) {
```

Comment:
java class `org.apache.hudi.common.util.collection.Pair` is preferred here over Tuple2

Reply:
> java class `org.apache.hudi.common.util.collection.Pair` is preferred here over Tuple2

![image](https://user-images.githubusercontent.com/10494131/153163154-a2a567da-5c62-450e-9ed7-87ac3223409a.png)

The methods exposed here require an array of Tuple2 as input parameters.